docs/quickstart.rst

```diff
@@ -0,0 +1,103 @@
+.. _quickstart:
+
+Pangeo Guide for Scientists
+===========================
+
+The goal of Pangeo is to make it quicker and easier to go from raw data to
+scientific insights.
+We want to spend less time writing boilerplate code and be able to iterate
+calculations quickly on very large datasets.
+These are ambitious yet achievable long-term goals.
+Reaching them will require scientists to participate in the open source process
+and contribute to the development of the underlying tools.
+For people coming from proprietary computing environments such as MATLAB, this
+can mean changing the way you think about software and your relationship to it.
+
+Below is a list of steps you can take to participate in the Pangeo project and
+take advantage of the tools we are developing.
+
+Learn About the Open Source Ecosystem
+-------------------------------------
+
+If you are new to open source scientific software, you might want to learn a
+bit about the history and context of this community.
+We recommend the following talk by `Jake VanderPlas <http://vanderplas.com/>`_
+entitled *PyData 101*.
+
+.. raw:: html
+
+   <iframe width="560" height="315" src="https://www.youtube.com/embed/DifMYH3iuFw" frameborder="0" allow="autoplay; encrypted-media" allowfullscreen></iframe>
+
+Jakes slides are `available on SpeakerDeck <https://speakerdeck.com/jakevdp/intro-to-pydata>`_.
+
+
+GitHub_ has become absolutely central to the way this community works.
+The Pangeo project is a GitHub-native organization.
+A great general-audience introduction to the power and popularity of GitHub
+can be found in the Wired article
+`The GitHub Revolution: Why We're All in Open Source Now <https://www.wired.com/2013/03/github/>`_
+by Mikeal Rogers.
+
+There are lots of great free and open resources to learn how to use scientific
+python.
+Here is a list of a few of our favorites.
+
+- list of
+- scientific python
+- online tutorials
```

Comment:
What are people's favorite online resources for teaching scientific python?

Reply:
Not limited to python but great as a concise set of rules to work by: The ten scientific coding commandments - - http://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1003285

And not strictly scientific python but great for getting people to understand concepts / ideas: The video tutorials by @CoreyMSchafer - - https://www.youtube.com/user/schafer5